.github/workflows/ci.yml

```
@@ -42,6 +42,7 @@ jobs:
 
       - uses: actions/setup-go@v6
         with:
+          go-version: "1.22"
           go-version: ${{ env.GO_VERSION }}
```

actionlint (1.7.12)

[error] key "go-version" is duplicated in "with" section. previously defined at line:45,col:11. note that this key is case insensitive

(syntax-check)